src/components/batch/BatchSidebar/BatchTxItem.tsx

```diff
@@ -26,6 +26,7 @@ const BatchTxItem = ({ id, count, timestamp, txDetails, onDelete }: BatchTxItemP
       txInfo: txDetails.txInfo,
       txStatus: txDetails.txStatus,
       safeAppInfo: txDetails.safeAppInfo,
+      txHash: txDetails.txHash || null,
```

Comment:
note: I'm adding this because the gateway SDK was updated for another PR which hasn't been merged yet #3772